cmd/releasego/get-images-commit.go

```diff
@@ -36,6 +37,9 @@ func handleGetImagesCommit(p subcmd.ParseFunc) error {
 	azdoVarName := flag.String("set-azdo-variable", "", "An AzDO variable name to set to the commit hash using a logging command.")
 	keepTemp := flag.Bool("w", false, "Keep the temporary repository used for polling, rather than cleaning it up.")
 	pollDelaySeconds := flag.Int("poll-delay", 5, "Number of seconds to wait between each poll attempt.")
+	ghClientId := githubutil.BindClientIDFlag()
```

Comment:
@dagood I was actually unsure about this file. As far as I can tell it's just running a `git fetch` and a `git show` command.

Currently we inject the GH PAT into the url like this:

```
-repo 'https://_:$(GitHubPAT)@github.com/$(TargetGoImagesGitHubRepo)' \
```

but I'm not sure it's actually necessary to pass the token to this command as the go-images repo is public right?

Reply:
got it, I'll leave the app support in place here then